pkg/kubelet/kuberuntime/instrumented_services.go

```diff
@@ -272,6 +272,15 @@ func (in instrumentedRuntimeService) PortForward(ctx context.Context, req *runti
 	return resp, err
 }
 
+func (in instrumentedRuntimeService) UpdatePodSandboxResources(ctx context.Context, req *runtimeapi.UpdatePodSandboxResourcesRequest) (*runtimeapi.UpdatePodSandboxResourcesResponse, error) {
+	const operation = "update_podsandbox"
```

Comment:
Should it be `update_podsandbox_resources`?

Reply:
From the [original update_container PR](https://github.com/kubernetes/kubernetes/pull/75278):

> Note that this uses update_container instead of update_container_resources as that is what the corresponding docker instrumented operation reports.

This isn't relevant here, so I'd prefer to go with `update_podsandbox_resources`. We should probably consider changing the update_container one too, as the docker parity is no longer relevant (not for this PR though).